lib/utils/regexes.mjs

```diff
@@ -5,6 +5,7 @@ export const atRuleRegexes = {
 	keyframes: /^(-(o|moz|ms|webkit)-)?keyframes$/i,
 	property: /^property$/i,
 	import: /^import$/i,
+	syntaxName: /^syntax$/i,
```

Comment:
Let's also move this out to a `descriptorRegexes` object:

```js
export const descriptorRegexes = {
  syntaxName: /^syntax$/i,
}
```
  
To remove any ambiguity in thinking it targets an at-rule named `@syntax`.

Reply:
Oh sorry, forgot to fix.

Reply:
Fixed in this commit!
[Refactor move syntaxName to descriptorRegexes](https://github.com/stylelint/stylelint/pull/8331/commits/f539267b15bd6f23456217f8f932b009bd4166f7)